.github/workflows/detect-renamed-docs.yml

```diff
@@ -0,0 +1,34 @@
+name: 🚀 Deploy
```

Comment:
I think this should be removed, I believed it escaped from the other PR you made. I left some comments on that, so deleting this you will avoid potential conflicts.

Reply:
Absolutely, it shouldn't be here. Thanks for noticing